src/python/pythoninterpretermodule.cpp

```diff
@@ -41,6 +41,8 @@ Status PythonInterpreterModule::start(const ovms::Config& config) {
         print(sys.version)
         print("Python executable:")
         print(sys.executable)
+        print("Python path:")
```

Comment:
```suggestion
        print("Python sys.path output:")
```
Let's make it as clear as possible